### PR TITLE
[AMO-128509] - Added Schema for Ad Cloud Clicks Data

### PR DESCRIPTION
**Pull request by jishah-adobe**
Please link to the issue #…

### CODE REVIEW

**Comment by jishah-adobe** [comment]
Hi @prabhum2 @cmathis @cfraser 

This PR is to define classes for search clicks data.  To give you a background, the lowest granularity of data collection mechanism is at ad/keyword level which is defined in Ad Performance Report Measure Schema.  The other two is basically a rollup of the Ad Performance Report Schema - one at the Ad Level and other at the Keyword Level.  If you have any questions please feel free to reach out to me

File: extensions/adobe/experience/adcloud/searchaggregatedperformancebyad.schema.json [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "https://ns.adobe.com/experience/adcloud/searchaggregatedperformancebyad",
+  "title": "Search Advertising Aggregated Performance Measures By Ad",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:extends": ["https://ns.adobe.com/xdm/data/time-series"],
+  "description": "Search Advertising Aggregated Performance Measures By Ad is a collection of aggregated measures pulled from Search Advertising Platforms by Ad/Keyword.",
+  "definitions": {
+    "searchaggregatedperformancebyad": {
```
Comment (udayeyunni) [inline extensions/adobe/experience/adcloud/searchaggregatedperformancebyad.schema.json]: Does XDM mandate all class names need to be in lower case? Makes it easy to read if we can use camelCase

File: extensions/adobe/experience/adcloud/searchaggregatedperformancebyadkeyword.schema.json [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "https://ns.adobe.com/experience/adcloud/searchaggregatedperformancebyadkeyword",
+  "title": "Search Advertising Aggregated Performance Measures By Ad and Keyword",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:extends": ["https://ns.adobe.com/xdm/data/time-series"],
+  "description": "Search Advertising Aggregated Performance Measures by Ad and Keyword is a collection of measures pulled from Search Advertising Platforms.",
+  "definitions": {
+    "searchaggregatedperformancebyadkeyword": {
```
Comment (udayeyunni) [inline extensions/adobe/experience/adcloud/searchaggregatedperformancebyadkeyword.schema.json]: Rename as "byad**By**keyword

**Comment by jishah-adobe** [comment]
Closing this in lieu of https://github.com/adobe/xdm/pull/703

**Pull request closed without merging**